forms/form/src/main.rs

```diff
@@ -186,7 +186,7 @@ mod tests {
             name: "John".to_string(),
         });
         let result = handle_post_3(req.clone(), params).await;
-        let resp = result.respond_to(&req);
+        let resp = result.customize().respond_to(&req);
```

Comment:
why is this necessary? no customization is being done.

Reply:
I'm sorry, this might be a beginner question, but I am getting this error.
How can it be suppressed?

<img width="711" src="https://user-images.githubusercontent.com/22282293/223754620-c7898028-d0d5-4c4e-8ef8-a661aaee007a.png">

Reply:
Thanks for the review. Then I will leave it untouched in this pull request.